spring-boot-project/spring-boot-docs/src/main/asciidoc/deployment.adoc

```diff
@@ -35,17 +35,23 @@ Once you have unpacked the jar file, you can also get an extra boost to startup
 
 More efficient container images can also be created by copying the dependencies to the image as a separate layer from the application classes and resources (which normally change more frequently).
 There is more than one way to achieve this layer separation.
-For example, using a `Dockerfile` you could express it in this form (assuming the jar is already unpacked at `target/dependency`):
+For example, using a `Dockerfile` you could express it in this form:
 
 [indent=0]
 ----
-	FROM openjdk:8-jdk-alpine
-	VOLUME /tmp
-	ARG DEPENDENCY=target/dependency
-	COPY ${DEPENDENCY}/BOOT-INF/lib /app/lib
-	COPY ${DEPENDENCY}/META-INF /app/META-INF
-	COPY ${DEPENDENCY}/BOOT-INF/classes /app
-	ENTRYPOINT ["java","-cp","app:app/lib/*","com.example.MyApplication"]
+    FROM openjdk:8-jdk-alpine AS builder
+    WORKDIR target/dependency
+    COPY ./target/*.jar .
+    RUN jar -xf ./*.jar
+
+    FROM openjdk:8-jre-alpine
+    EXPOSE 8080
```

Comment:
I am not really an export but I don't understand why hardcoding 8080 here is something we should do. This might not be a webapp. It might be a webapp running on a different port.

Reply:
My bad, not need at all. 
It's some rest of local testing, sorry 😕

Reply:
By the way the main point here was to use a jre instead of a jdk alpine image